core/dictgen/src/rootcling_impl.cxx

```diff
@@ -3409,6 +3409,9 @@ std::list<std::string> RecordDecl2Headers(const clang::CXXRecordDecl &rcd,
 {
    std::list<std::string> headers;
 
+   // We push a new transaction because we could deserialize decls here
+   cling::Interpreter::PushTransactionRAII RAII(&interp);
```

Comment:
Do we know/understand which part of the code can deserialize?  I.e. Can this be safely move to line 3422?